buildenv/jenkins/JenkinsfileBase

```diff
@@ -23,18 +23,20 @@ def makeTest(testParam) {
 def setupEnv() {
 	// if ADOPTOPENJDK_REPO and ADOPTOPENJDK_BRANCH are provided, clean the workpsace and re-download it
 	if (params.ADOPTOPENJDK_REPO && params.ADOPTOPENJDK_BRANCH) {
-		cleanWs()
+		if ( !params.ADOPTOPENJDK_REPO.contains( 'AdoptOpenJDK/openjdk-tests.git') || params.ADOPTOPENJDK_BRANCH != "master") {
```

Comment:
Could we also update the comment in line 24?

Reply:
and fix spelling workspace instead of workpsace

Reply:
The comment in line 24 has beed updated, with fixing "workspace" spelling. Thanks.